beacon-chain/sync/service_test.go

```diff
@@ -0,0 +1,48 @@
+package sync
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/internal"
+)
+
+func TestStatus(t *testing.T) {
+
+	db := internal.SetupDB(t)
+	defer internal.TeardownDB(t, db)
+
+	if err := db.InitializeState(); err != nil {
+		t.Fatalf("Failed to initialize state: %v", err)
+	}
+
+	cfg := &Config{
+		ChainService:  &mockChainService{},
+		P2P:           &mockP2P{},
+		BeaconDB:      db,
+		AttestService: &mockAttestService{},
+	}
+
+	ss := NewSyncService(context.Background(), cfg)
+	synced, err := ss.Querier.IsSynced()
```

Comment:
call this err `querierErr` and the other one `statusErr` for clarity

Reply:
Done!